internal/encoding/block/from_parquet.go

```diff
@@ -92,3 +99,49 @@ func FromParquetBy(payload []byte, partitionBy string, filter *typeof.Schema, ap
 	blocks = append(blocks, last...)
 	return blocks, nil
 }
+
+type fieldHandler func(interface{}) (interface{}, error)
+
+func getReverseHandler(typ string) (fieldHandler func(interface{}) (interface{}, error), err error) {
```

Comment:
A better name would be `parquetHandlerFor(typ string) parquetHandler`. We don't need to return an error here, since you're ignoring it anyway when using it and never return here.